assets/embauche.html

```diff
@@ -321,6 +299,6 @@
 			</tr>
 		</table>
 
-		<button id="createTest">Corriger ces résultats</button>
+		<button id="reportDetailsProblem"><a href="mailto:embauche@sgmap.fr?subject=Erreur dans les résultats du simulateur">Signaler une erreur</a></button>
```

Comment:
`<button>` should be [wrapped by](http://stackoverflow.com/questions/16280684) `<a>`. Or, even better, make it simply an `<a>` and style it with CSS to give it the look of a button if you want to have it look like a button.